grammars/less.cson

```diff
@@ -277,7 +277,7 @@
     'name': 'keyword.operator.less'
   }
   {
-    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|when)\\s'
+    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|(when not)|when)\\s'
```

Comment:
1. Yes `when(\\s+not)?` is more proper to group `when_____not` as one token.
2. I will do cleanup.